doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -267,7 +267,8 @@ nRF5340 Audio
 * Updated:
 
   * ISO data sending has been refactored, and is now done in a single file: bt_le_audio_tx.
```

Comment:
This and other edits in this section are addressing the changelog formatting, where "Updated" needs to match the sentences in the bullets.
```suggestion
  * Sending of the ISO data, which is now done in a single file :file:`bt_le_audio_tx`.
```

Reply:
This is strictly not part of this PR, same for the comment below

Reply:
OK, I'll open a separate PR...

Reply:
Don't need to, I've changed them here now.